### DISCUSSION TITLE
Sequence Stops and Delay in FPP Start (12/28/2023 build)

### DISCUSSION THREAD

**Discussion by DShaeffer** [forkineye/ESPixelStick#722] [Q&A]
This is my first year playing with pixels. I am using the CI build that was pushed yesterday on an ESPixelStick because I was having issues with my sequence file playing on v4-beta5. The sequence now run three or four times and then it just stops and the ESPixelStick must be rebooted to work again. Also, there is always a 8-13 second delay from the FPP starting the sequence and the ESP playing. How can I best troubleshoot these issues? The last error in the FFP status is blank.

**Comment by MartinMueller2003** [comment]
Send a screenshot of the admin page.
Turn on admin mode
Send a screenshot os the status page after the file stops playing.
Attach the fseq file to this thread. 
You can get the latest build here: https://github.com/MartinMueller2003/ESPixelStick/releases/download/dist/dist.zip. Please make sure this is what you are testing with.

**Comment by DShaeffer** [comment]
The dev build keeps playing which is great. I am still seeing a ~13 second delay when restarting. How do I prevent that? Both the Raspberry Pi with FPP and the ESPixelStick have have a strong wifi connection.

**Comment by DShaeffer** [comment]
It finally did stop playing after maybe a dozen times and required a reboot. 
![image](https://github.com/forkineye/ESPixelStick/assets/5730673/8b46c98e-2738-47db-a711-2cfa55fde102)
[animationplaceholder.zip](https://github.com/forkineye/ESPixelStick/files/13801957/animationplaceholder.zip)

**Comment by MartinMueller2003** [comment]
I have a 1 minute test sequence.
If I play it with the UI on the status page, it runs for about an hour and then the TCP library crashes. Nothing much I can do for that. If I run it with the UI turned off (no browser active, I ran for five hours without crashes. 
I started and stopped the FPP multiple times and the ESP responded faster than I could see. 

Test:
Test file is loaded on a V7 series FPP and the latest ESP32 build for a dig quad.
FPP plays the file in a loop using multicast.
The test file is outputting 1000 pixels worth of data at a 25ms refresh rate.

I will move next to testing using your file.

**Comment by MartinMueller2003** [comment]
I played your sequence for 5 hours and am now playing my show without any issues. Show uses FPP 5.5 so it is now tested with both versions. I did have a crash while configuring, but that is not the issue you described.

**Comment by MartinMueller2003** [comment]
<img width="815" alt="image" src="https://github.com/forkineye/ESPixelStick/assets/8608503/b66d34c8-2b75-4014-a931-29d3c3fa5ebe">

**Comment by MartinMueller2003** [comment]
I just finished a test where FPP played a fill 10360 times before I killed the test.
